src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -1,12 +1,21 @@
 import { pxToRem } from '../../../../lib'
 import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
 import { IListItemProps } from '../../../../components/List/ListItem'
+import isFromKeyboard from '../../../../lib/isFromKeyboard'
 
 const listItemStyles: IComponentPartStylesInput<IListItemProps, any> = {
   root: ({ props: { selection, important } }): ICSSInJSStyle => ({
     ...(selection && {
       position: 'relative',
 
+      ':focus': {
+        ...(isFromKeyboard && {
+          background: 'rgba(98, 100, 167, .8)',
+          color: '#fff',
+        }),
+        outline: 0,
+      },
+
       ':hover': {
         background: 'rgba(98, 100, 167, .8)',
```

Comment:
looks like we need a color for this, can we find the hex value in teams colors and add it to `siteVariables.ts` if not there?

Reply:
that is already done in https://github.com/stardust-ui/react/pull/278 so I would wait for that one to be merged